homeassistant/components/roon/media_player.py

```diff
@@ -481,31 +475,27 @@ def set_shuffle(self, shuffle):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the play_media command to the media player."""
-        # Roon itself doesn't support playback of media by filename/url so this a bit of a workaround.
-        media_type = media_type.lower()
-        if media_type == "radio":
-            if self._server.roonapi.play_radio(self.zone_id, media_id):
-                self._last_playlist = media_id
-                self._last_media = media_id
-        elif media_type == "playlist":
-            if self._server.roonapi.play_playlist(
-                self.zone_id, media_id, shuffle=False
-            ):
-                self._last_playlist = media_id
-        elif media_type == "shuffleplaylist":
-            if self._server.roonapi.play_playlist(self.zone_id, media_id, shuffle=True):
-                self._last_playlist = media_id
-        elif media_type == "queueplaylist":
-            self._server.roonapi.queue_playlist(self.zone_id, media_id)
-        elif media_type == "genre":
-            self._server.roonapi.play_genre(self.zone_id, media_id)
-        elif media_type in ("library", "track"):
-            self._server.roonapi.play_id(self.zone_id, media_id)
-        else:
+        # media_id is treated as a path matching the Roon menu structure
+
+        def splitall(path):
```

Comment:
This should probably be a helper in the library that we can import and use here.

Reply:
Added to the library. 

And added the version bump (and version change into to the PR)